content/java/concepts/math-methods/terms/to radian/to-radian.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '.toRadians()'
+Description: 'Converts an angle in degrees to an approximately equivalent angle in radians.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.toRadians`** method returns the measurement of an angle in degrees to an equivalent angle in radians.
+
+## Syntax
+
+```pseudo
+Math.toRadians(deg)
+```
+
+The `deg` argument is a `double` value.
+The conversion from degrees to radians is generally inexact.
+
+## Example
+
+The following example uses `Math.toRadians()` to return radian measurement of 180 and 45 degrees:
```

Comment:
~The following example uses `Math.toRadians()` to return radian measurement of 180 and 45 degrees:~ 

Consider the following sentence instead:

The following example uses `Math.toRadians()` to convert 180 degrees and 45 degrees to radians: